Makefile

```diff
@@ -0,0 +1,11 @@
+.PHONY: all image test
+
+IMAGE_NAME=codeclimate/codeclimate-duplication
+
+all: image test
+
+image:
```

Comment:
Hmm. It's worth questioning, but I'm not sure I share your concern:

- An existing project with an existing Makefile is unlikely to be using `image`, IMHO. While it is generic, I've never seen `make image` in the wild
- The intent of this target is (to me) not specifically to "make a codeclimate engine", but to build a docker image named `IMAGE_NAME` -- `make image` seems very apt

For those reasons, I'd be OK going with this for now. Could be worth sleeping on it though.